stats.go

```diff
@@ -54,6 +68,11 @@ func GetStats() (snapshot Stats) {
 	return
 }
 
+// GetStatsEvents returns the struct containing event channels
+func GetStatsEvents() *StatsEvents {
+	return statsEvents
+}
+
 // ResetStats reset Stats to the previous database state
 func ResetStats() {
```

Comment:
After the library user calls ResetStats(), GetStatsEvents() needs to be called immediately, at least from my understanding of the intended usage. 

Why not return new stats from the ResetStats() func. It can always be ignored.

Reply:
My understanding is that this would be a breaking change - someone might be passing the `ResetStats` function somewhere that expects a `func()` callback type or something, for example.

It might be better to just decouple the StatsEvents thing from the Stats thing entirely, and require an explicit call to `SetStatsEvents(bool)` or something to turn it on and off (and don't provide a reset function at all - doesn't really seem necessary)